File: conditionals/time.py
#samantha naranjo, time of day py
import time

#find the first instance of time in programming

first_time = time.gmtime()
#print(first_time) # type: ignore

#current time in seconds
current = time.time()
#print(current) #seconds since jan 1 1970

#cirrent date and time we see it normally
now = time.ctime(current)
#print(now)

#get just a part of the time
local_time = time.localtime(current)
day = local_time.tm_wday
hour = local_time.tm_wday #military time (0-23)
print(hour)

File: conditionals/time.c
//samantha naranjo, getting the time
#include <stdio.h>
#include <time.h>

int main(void){
   time_t seconds
   seconds = time(NULL);
   printf("seconds since jan 1, 1970 = %d\n", seconds);

   //current time 
   time_t rawtime;
   struct tm * timeinfo

   //time(&rawtime);
   //timeinfo = localtime(&rawtime);
  //printf("current time and date us %s\n ", asctime(timeinfo));
   
   //curent hour
    time_t now = time(NULL);
    
    struct tim * tm_struct = localtime (&now);
    int hour = tm_struct->tm_hour;
    printf("%d\n", hour);

   return 0;

}